<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resultados</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 246, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>RESULTADOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>GRADO DE COMPLICACION: </string>
   </property>
  </widget>
  <widget class="QLabel" name="resgrado">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>70</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SINTOMAS QUE PRESENTA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ressintomas">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>122</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="consultar1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>lungs-2803208_960_720.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="correo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>@gmail.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@hotmail.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@udea.edu.co</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingrese el correo del paciente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TRATAMIENTO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tratamiento_2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>190</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enviar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="correo1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
